flink-kubernetes-shaded/pom.xml

```diff
@@ -69,6 +81,16 @@ under the License.
                                     <shadedPattern>org.apache.flink.kubernetes.shaded.io.fabric8</shadedPattern>
                                 </relocation>
                             </relocations>
+                            <filters>
+                                <filter>
+                                    <artifact>*:*</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/DEPENDENCIES</exclude>
+                                        <exclude>META-INF/LICENSE</exclude>
+                                        <exclude>META-INF/MANIFEST.MF</exclude>
+                                    </excludes>
+                                </filter>
+                            </filters>
```

Comment:
This will remove the NOTICE / LINCENSE files from the shaded jar which is probably not right from a licensing perspective as we are releasing this artifact. 

What do you think @wangyang0918